livekit-agents/livekit/agents/voice/agent_session.py

```diff
@@ -273,17 +276,26 @@ async def start(
             self._started = True
             self._update_agent_state(AgentState.LISTENING)
 
-    async def aclose(self) -> None:
+    async def aclose(self, cause: ErrorEvent | None = None) -> None:
```

Comment:
Since this is a user facing API, I doin't think it should have any parameter, let's create an internal aclose_impl instead?